src/KafkaNET.Library/Consumers/PartitionTopicInfo.cs

```diff
@@ -232,8 +232,8 @@ public int Add(BufferedMessageSet messages)
 
                 Logger.InfoFormat("{2} : Updating fetch offset = {0} with value = {1}", this.fetchedOffset, offset, this.PartitionId);
                 this.chunkQueue.Add(new FetchedDataChunk(messages, this, this.fetchedOffset));
-                long newOffset = Interlocked.Exchange(ref this.fetchedOffset, offset);
-                Logger.Debug("Updated fetch offset of " + this + " to " + newOffset);
```

Comment:
@mhorstma while editing this file, I had looked at what FetchedDataChunk does with the fetchedOffset. I don't recall all very well, but the order seemed correct. Let us open another PR if we find bugs.

The logging was misleading because the Interchange.Exchange() returns the old value, and not the new offset value. So, I removed the "long newOffset" variable altogether. The debug logs should now be true :)